Ant.cpp

```diff
@@ -1,203 +1,97 @@
-** Program name:Group Project - Predator-Prey Game
+** Program Name:Group Project - Predator-Prey Game
 ** Author:	Group 4
+**		      Herbert Diaz  <diazh@oregonstate.edu>
+**          Janet Anderson <andejane@oregonstate.edu>
 ** Date:	2019/02/04
 ** Description: THIS IS THE IMPLEMENTATION FILE OF THE ANT CLASS
-**              This program runs a Predator-Prey Game with "Doodlebug"
-**              predators and "Ant" prey. The program starts with a menu and
-**              has the user set the number steps for the simulation to take.
-**              Once the simulation is complete, the user is queried for the
-**              whether they want to continue with more steps or quit.
+**		          It contains the default constructor creating an Ant object and the move function
+**              that will move the Ant object on the board based on randomly selecting a cardinal
+**              direction and checking that the move would not push the ant off the board and the
+**              space is not occupied before moving.
 **************************************************************************************************/
-// Ant.cpp
-
 #include "Critter.hpp"
 #include "Ant.hpp"
 #include "Group4Utils.hpp"
 
 /**************************************************************************************************
-** Description: Ant()
-**              This is the default constructor for the Ant Class
+** Description: Ant(int,int)							 **PUBLIC::2019.02.04::HD-JA**
+**              This is the default constructor for the Ant Class that sets the breeding age of the 
+**              Ant to 3, the board symbol representing the Ant to 'O', and the moveSuccess to 0.
 **************************************************************************************************/
 Ant::Ant(int x_pos, int y_pos):Critter(x_pos, y_pos)
 {
    this->breedAge = 3;
    this->symbol = 'O';
+   this->moveSuccess = 0;
 }
 
-
 /**************************************************************************************************
-** Description: void move(Critter***, int, int))
-**              This function moves an Ant on the board.
-**              The Critter*** argument represents the board the Critters are on.
-**              The two integers represent the dimensions of the board.
+** Description: void move(Critter*** &, int, int))		   **PUBLIC::2019.02.04::HD-JA-CC**
+**		          The move function for the Ant class will randomly call a number between 0-3, which
+**              represents the cardinal directions (NORTH, EAST, SOUTH, and WEST).  Based on the
+**              random direction, the ant will attempt to move to that location as long as the 
+**              move will not go out of bounce and the space is unoccupied.
 **************************************************************************************************/
 void Ant::move(Critter *** &boardIn, int size_x, int size_y)
 {
-      int new_x, new_y;
```

Comment:
Have to make sure these changes are based on the fixed version of the move function made by Jessica

Reply:
It should be, I cleaned it up to make it more efficient and to make sure we were only attempting to move once during the move function, but the actual movement code should be same.